.dockerignore

```diff
@@ -1 +1,2 @@
 flink-sample-job/
+!flink-sample-job/target/scala-2.11/flink-stateful-wordcount-assembly-0.jar 
```

Comment:
Isn't it better if we exclude the entire target folder?

Reply:
yes good point, would support potential future versions of sample-job